Source/WebInspectorUI/UserInterface/Views/MultiSidebar.js

```diff
@@ -43,6 +43,7 @@ WI.MultiSidebar = class MultiSidebar extends WI.Sidebar
         this._requiredMinimumWidthForMultipleSidebars = 0;
 
         window.addEventListener("resize", this._handleWindowResize.bind(this));
+        WI.notifications.addEventListener(WI.Notification.LayoutModeDidChange, this._adjustPosition, this);
```

Comment:
Event handlers in Web Inspector usually contain the event name for clarity and to aid mental mapping.

`this._adjustPosition` -> `this._handleLayoutModeDidChange`

Reply:
Done.